README.md

```diff
@@ -18,9 +18,12 @@ The commands below assume you have a shell alias setup in your .bashrc or .zshrc
 -   Run `sail artisan migrate`
 -   Run `sail npm install`
 -   Run `sail npm run dev`
+-   Run `sail artisan db:seed`
```

Comment:
Do we need to run seed in general or just `artisan db:seed --class=DefaultAdmin`? Will `sail artisan db:seed` seed other things too?

Reply:
sail artisan db:seed will seed all the test data. 

because i dont want to fill a prod database with test data, i saved this as a different seeder file and didn't add it to php artisan db:seed. perhaps we add it there, but not the other way around.. that makes sense